tests/integration_web/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Juan Alvarez <j@yabit.io>"]
 edition = "2021"
-name = "integration"
+name = "integration_web"
```

Comment:
I'm not a fan of the `integration_` prefix here. Everything in `/tests` is an integration test by default, and adding the prefix here doesn't seem useful -- just call it `/tests/web`?

Reply:
Changed to `/tests/web`.